include/flatbuffers/flatbuffers.h

```diff
@@ -75,6 +75,9 @@
 
 // The wire format uses a little endian encoding (since that's efficient for
 // the common platforms).
+#if defined(__s390x__)
+  #define FLATBUFFERS_LITTLEENDIAN 0
+#endif // __s390x__
 #if !defined(FLATBUFFERS_LITTLEENDIAN)
   #if defined(__GNUC__) || defined(__clang__)
     #ifdef __BIG_ENDIAN__
```

Comment:
Ok, looks like no endian define is on by default: https://www.ibm.com/developerworks/library/l-systemz/
So we'll merge this as-is.